src/remote-config.ts

```diff
@@ -54,6 +57,9 @@ export const defaultRemoteConfig: RemoteConfig = {
   tariff_zones: JSON.stringify(defaultTariffZones),
   user_profiles: JSON.stringify(defaultUserProfiles),
   customer_service_url: 'https://www.atb.no/kontakt/',
+  tripsSearch_target_number_of_initial_hits: 3,
+  tripsSearch_target_number_of_page_hits: 10,
+  tripsSearch_max_number_of_chained_searches: 5,
```

Comment:
Ser bra ut dette! Gode navn, og lurt å dele opp i initial- og page-hits :+1:

Når det gjelder selve verdiene så kanskje vi skulle tatt en kjapp runde med Hilde/Simen, og eventuelt andre i teamet? Hvis jeg prøver å tenke hva som er intuitivt for meg så er det at det første søket gir flere eller like mange reiseforslag som det man får ved paginering. Vi viser uansett søkeresultater etter hvert som de oppstår, både ved første søk og paginering, så tweakingen kan gjøres i hovedsak basert på hva vi forventer gir best brukeropplevelse.